pubpub/pubpub
# fix: default pub connection view to list

This PR restores the previous list view default for the Connection listing at the bottom of Pubs.

_Test Plan_
1. Create a Pub with at least two Connections.
2. Ensure the "Show multiple Connections as" radio group is set to "list" by default.
3. Open the Pub and confirm that the Connection listing defaults to the list view.
